packages/block-library/src/post-comments-form/edit.js

```diff
@@ -62,13 +62,10 @@ export default function PostCommentsFormEdit( {
 
 	if ( ! isSiteEditor && 'open' !== commentStatus ) {
 		if ( 'closed' === commentStatus ) {
-			warning = sprintf(
-				/* translators: 1: Post type (i.e. "post", "page") */
-				__(
-					'Post Comments Form block: Comments on this %s are not allowed.'
-				),
-				postType
+			warning = __(
+				'Post Comments Form block: Comments are not enabled for this item.'
```

Comment:
I used `enabled` rather than `allowed` to be more consistent with the [button label](https://github.com/WordPress/gutenberg/pull/42131/files#diff-abf88ae09f89f6af7894b1f8480350eb820df74aff7e2b61490e93dd48c4722fR76) and [other](https://github.com/WordPress/gutenberg/pull/42131/files#diff-abf88ae09f89f6af7894b1f8480350eb820df74aff7e2b61490e93dd48c4722fR86) [messages](https://github.com/WordPress/gutenberg/pull/42131/files#diff-abf88ae09f89f6af7894b1f8480350eb820df74aff7e2b61490e93dd48c4722fR93).

Reply:
I also changed the word order a bit since `Comments for this item are not enabled` sounded a bit awkward to me.